toolchains/toolchains.bzl

```diff
@@ -79,6 +80,17 @@ current_ninja_toolchain = rule(
     ],
 )
 
+current_meson_toolchain = rule(
+    implementation = _current_toolchain_impl,
+    attrs = {
+        "_toolchain": attr.string(default = str(Label("//toolchains:meson_toolchain"))),
```

Comment:
I suspect this code will need to change to work with bzlmod but we can address that at a later date.